ui/app/src/modules/Preview/Tools/AudiencesPanel.tsx

```diff
@@ -15,24 +15,250 @@
+    onSetDefaults
+  } = props;
+  const contentType = audiencesResource.read();
+
   return (
     <ToolPanel title={translations.audiencesPanel}>
-      <Typography component="h2" variant="subtitle1" style={{ padding: '10px' }}>
-        Audiences Panel
-      </Typography>
+      {
+        <>
+          <Grid className={classes.PanelMargin}>
+            {
+              Object.keys(contentType.fields).map((field: any) => {
+                const type = contentType.fields[field].type;
+                const Control = controlsMap[type];
+
+                const controlProps = {
+                  field: contentType.fields[field],
+                  value: model[field] ? model[field].key : undefined,
+                  onChange: onFormChange,
+                  disabled: modelApplying
+                };
+
+                if (controlProps.field === 'date-time' && model[`${field}_tz`]) {
+                  controlProps['timezone'] = model[`${field}_tz`].key;
+                }
+
+                return (
+                  <AudiencesFormSection field={contentType.fields[field]} key={field} showDivider={true}>
+                    <Control
+                      {...controlProps}
+                    />
+                  </AudiencesFormSection>
+                )
+              })
+            }
+          </Grid>
+          <Grid className={classes.actionBTN}>
+            <Button variant="contained" onClick={() => onSetDefaults(contentType)}>
+              <FormattedMessage
+                id="audiencesPanel.defaults"
+                defaultMessage={`Defaults`}
+              />
+            </Button>
+            <Button variant="contained" color="primary" onClick={() => onSaveModel()}>
+              <FormattedMessage
+                id="audiencesPanel.apply"
+                defaultMessage={`Apply`}
+              />
+            </Button>
+          </Grid>
+        </>
+      }
     </ToolPanel>
   );
 }
+
+export default function AudiencesPanel() {
+  const state = useSelection<GlobalState['preview']['audiencesPanel']>(state => state.preview.audiencesPanel);
+  const resource = useStateResource(
+    state,
+    {
+      shouldRenew: (source, resource) => resource.complete && nou(source.contentType),
+      shouldResolve: source => (!source.isFetching) && nnou(source.contentType) && nnou(source.model),
+      shouldReject: source => nnou(source.error),
+      errorSelector: source => source.error,
+      resultSelector: source => source.contentType
+    }
+  );
+
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+
+  const onFormChange = (name: string, value: string) => {
+    dispatch(updateAudiencesPanelModel({
+      [name]: {
+        key: value,
+        label: value
+      }
+    }));
+  };
+
+  const saveModel = () => {
+    const model = reversePluckProps(state.model, 'craftercms');
+    const params = encodeURI(Object.entries(model).map(([key, val]) => `${key}=${val.key}`).join('&'));
```

Comment:
Not the component‘s responsibility to serialise. This would be done by the service.

In general, speak in JS and let services translate to server jargon. In some cases epics may apply some transformations before calling the service too.